### PR TITLE
chore(genesis): Add missing account to genesis

### PR DESCRIPTION
**Pull request by nddeluca**
This an account was added to genesis in #203 but not added to the generation script.

This adds the account and re-runs the genesis generation with the latest kava master binary.

Depends on #204

### CODE REVIEW

File: config/templates/kava/master/initstate/.kava/config/genesis.json [hunk 1]
```diff
@@ -807,15 +816,6 @@
               "amount": "10000000000000000"
             }
           ]
-        },
```
Comment (nddeluca) [inline config/templates/kava/master/initstate/.kava/config/genesis.json:810]: Order is changed, but should be consistent from here on out

File: config/templates/kava/master/initstate/.kava/config/genesis.json [hunk 2]
```diff
@@ -2221,7 +2221,8 @@
             "nested_types": []
           }
         ],
-        "allow_unprotected_txs": false
+        "allow_unprotected_txs": false,
+        "enabled_precompiles": []
```
Comment (nddeluca) [inline config/templates/kava/master/initstate/.kava/config/genesis.json:2225]: This is the default value from current kava master

File: config/templates/kava/master/initstate/.kava/config/app.toml [hunk 3]
```diff
@@ -62,6 +62,7 @@ index-events = []
 # IavlCacheSize set the size of the iavl tree cache (in number of nodes).
```
Comment (nddeluca) [inline config/templates/kava/master/initstate/.kava/config/app.toml:62]: Upstream change

File: config/templates/kava/master/initstate/.kava/config/gentx/gentx-461d9761df5db0a112407ca8a82f1774ef2dbaa3.json [hunk 4]
```diff
@@ -1 +1 @@
...
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"validator","identity":"","website":"","security_contact":"","details":""},"commission":{"rate":"0.100000000000000000","max_rate":"0.200000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"1","delegator_address":"kava1ypjp0m04pyp73hwgtc0dgkx0e9rrydecm054da","validator_address":"kavavaloper1ypjp0m04pyp73hwgtc0dgkx0e9rrydeckewa42","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"/iEtt5/G+7hUT4FJ4p29yi2DCRut5Bm1Zx6doestjUA="},"value":{"denom":"ukava","amount":"1000000000"}}],"memo":"461d9761df5db0a112407ca8a82f1774ef2dbaa3@172.16.231.128:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"AgIWiZpdEVb69gfaqyNjbA0b3oe+nRS9BZ9gH/I6oGz+"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""},"tip":null},"signatures":["ytsFwc8lookk5/jCWGjxkoNHOGgfuQx6ol2tzclZSMURpLe0s9oAzY7jGH7hS+om9jrR/b9YlfRXG1dHuzlPeg=="]}
```
Comment (nddeluca) [inline config/templates/kava/master/initstate/.kava/config/gentx/gentx-461d9761df5db0a112407ca8a82f1774ef2dbaa3.json:1]: This is normal to be updated every new genesis generation

File: config/templates/kava/master/initstate/.kava/config/genesis.json [hunk 5]
```diff
@@ -1,5 +1,5 @@
 {
-  "genesis_time": "2024-07-12T00:27:25.024459Z",
+  "genesis_time": "2024-08-07T19:18:19.347530324Z",
```
Comment (nddeluca) [inline config/templates/kava/master/initstate/.kava/config/genesis.json:2]: Time is normal to be updated w/ genesis generation

**Review by sesheffield — APPROVED** [review]

File: config/generate/genesis/generate-kava-genesis.sh [hunk 6]
```diff
@@ -264,6 +264,8 @@ add-eth-genesis-account-key bridge_relayer '.kava.users.bridge_relayer' 10000000
 add-genesis-account kava1cj7njkw2g9fqx4e768zc75dp9sks8u9znxrf0w 1000000000000ukava,1000000000000swp,1000000000000hard
 # swap module
 add-genesis-account kava1mfru9azs5nua2wxcd4sq64g5nt7nn4n8s2w8cu 5000000000ukava,200000000btcb,1000000000hard,5000000000swp,103000000000usdx
+# fractional balance test account
+add-genesis-account kava1w9vxuke5dz6hyza2j932qgmxltnfxwl78u920k "2ukava"
```
Comment (drklee3) [inline config/generate/genesis/generate-kava-genesis.sh:268]: Likely removing it soon after precisebank upgrade, but would be good to mention in the comment why it's 2ukava for clarity - must match total value of `evmutil.accounts[].balance`

**Review by drklee3 — APPROVED** [review]

**Pull request merged**